### PR TITLE
Tailwind v2 docs

### PR DESCRIPTION
**Pull request by delucis**
## Changes

- Add migration section to Tailwind v2 changelog
- Update README to include `tailwindcss` in manual install instructions
- Is there anywhere else we can/should document the migration steps?

## Testing
Docs only

## Docs

cc @withastro/maintainers-docs for feedback!

### CODE REVIEW

File: .changeset/swift-trees-double.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/tailwind": patch
```
Comment (matthewp) [inline .changeset/swift-trees-double.md:2]: do we need to do a release from this? People will just read this from the github anyways, right?

Comment (delucis) [reply .changeset/swift-trees-double.md:2]: This also updates the README which we usually trigger a patch for but I can remove if you think it's unnecessary.

**Review by sarah11918 — APPROVED** [review]
Docs updates approved, @delucis! Thanks for this!

**Review by matthewp — APPROVED** [review]
sgtm

**Comment by delucis** [comment]
Thanks all! I've also manually updated the GitHub release notes to match: https://github.com/withastro/astro/releases/tag/%40astrojs%2Ftailwind%402.0.0

**Pull request merged**